README.md

```diff
@@ -1 +1,29 @@
-# transmogrifiers
\ No newline at end of file
+# Transmogrifiers
+This repo contains the filters, standard library, and schema files to properly process public art json files retrieved from open data sets provided by municipalities.
```

Comment:
wouldn't just be public art and json, could be worded more generically, maybe reword as datasets?